images/linux/scripts/installers/1604/dotnetcore-sdk.sh

```diff
@@ -34,11 +34,6 @@ for latest_package in ${LATEST_DOTNET_PACKAGES[@]}; do
     echo "Determing if .NET Core ($latest_package) is installed"
     if ! IsInstalled $latest_package; then
         echo "Could not find .NET Core ($latest_package), installing..."
-        curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg
-        mv microsoft.gpg /etc/apt/trusted.gpg.d/microsoft.gpg
-        sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-ubuntu-bionic-prod bionic main" > /etc/apt/sources.list.d/dotnetdev.list'
-        apt-get install apt-transport-https
```

Comment:
Could you please confirm we don't really need `apt-transport-https` package?

Reply:
This package also will be installed in kubernetes-tools.sh
I think we don't need install it twice in Ubuntu16 build

Reply:
Does it matter when this is installed - earlier or later? Are we using it somehow during the image generation?